po/tr.po

```diff
@@ -412,17 +416,17 @@ msgstr "D-Bus hatası: %s"
 msgid "About gPodder"
 msgstr "gPodder hakkında"
 
-#: src/gpodder/gtkui/app.py:227
+#: src/gpodder/gtkui/app.py:226
 msgid "Website"
 msgstr "İnternet Sitesi"
 
-#: src/gpodder/gtkui/app.py:228
+#: src/gpodder/gtkui/app.py:227
 msgid "Bug Tracker"
 msgstr "Hata İzleyici"
 
-#: src/gpodder/gtkui/app.py:279
+#: src/gpodder/gtkui/app.py:277
 msgid "Path to gPodder home is too long"
-msgstr "gPodder eve giden yol çok uzun"
+msgstr "gPodder ana sayfasına giden yol çok uzun"
```

Comment:
not sure if it's correct: the message is used when the gpodder home folder (~/gPodder on Linux) is too deep in the folder hierarchy.
*Google translate* translates your translation back to "Path to gPodder homepage is too long", but they definitely could be wrong